stock_location_orderpoint/README.rst

```diff
@@ -0,0 +1,140 @@
+=========================
+stock_location_orderpoint
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--orderpoint-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-orderpoint/tree/16.0/stock_location_orderpoint
+    :alt: OCA/stock-logistics-orderpoint
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-orderpoint-16-0/stock-logistics-orderpoint-16-0-stock_location_orderpoint
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/stock-logistics-orderpoint&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Declare orderpoint on a location allowing to replenish any product with the same criteria.
+This is for an internal warehouse replenishment currently not compatible with the purchase buy route.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+First configuration
+===================
+
+#. In order to replenish your stock location from another one, you first need
+   to set the multi locations configuration.
+#. So, go to Inventory > Configuration > Settings > Warehouse
+#. Check the 'Storage Locations' box.
+#. As you should be able to configure a dedicated route to replenishment, you
+   should also activate, in the same menu, the 'Multi-Step Routes' box.
+
+Locations configuration
+=======================
+
+#. Identify the location you want to apply a replenishment rule (e.g.: WH/Stock)
+#. Create (if not exists) a new location for replenishment under Warehouse view (e.g.: WH)
+   location as we want to get the stock in replenishment taken into account of
+   our product stock total quantity.
+
+Route Configuration
+===================
+
+#. You should configure at least a route with a rule that:
+
+    * Pull from the Replenishment stock location.
+    * For the stock location you want to (e.g.: WH/Stock)
+
+Location Orderpoint configuration
+=================================
+
+#. Go either by the stock location you want to replenish and click on 'Orderpoints'
+   or go to Inventory > Configuration > Settings > Warehouse > Stock Location Orderpoint
```

Comment:
The menu route here is incorrect:

![image](https://github.com/OCA/stock-logistics-orderpoint/assets/23449160/f62049be-8c0b-467d-8899-8ff7d1c39e3a)

Also, you can see that there is a problem in the form view with the active toggle button

Reply:
thanks a lot i will fix this

Reply:
![image](https://github.com/OCA/stock-logistics-orderpoint/assets/78783475/238b2c23-abed-4931-85b5-c3d597206d3a)
![image](https://github.com/OCA/stock-logistics-orderpoint/assets/78783475/7e6df109-356c-4aff-86bd-651b2d6f66dd)
![image](https://github.com/OCA/stock-logistics-orderpoint/assets/78783475/8fd5cc31-e2c0-4595-95ac-ea4c44898457)

I added a ribbon